decidim-meetings/app/forms/decidim/meetings/admin/meeting_registrations_form.rb

```diff
@@ -11,12 +11,16 @@ class MeetingRegistrationsForm < Decidim::Form
 
         attribute :registrations_enabled, Boolean
         attribute :available_slots, Integer
+        attribute :reserved_slots, Integer
         translatable_attribute :registration_terms, String
 
         validates :registration_terms, translatable_presence: true, if: ->(form) { form.registrations_enabled? }
         validates :available_slots, numericality: { greater_than_or_equal_to: 0 }, if: ->(form) { form.registrations_enabled? }
-        validate :available_slots_greater_than_or_equal_to_registrations_count, if: ->(form) { form.registrations_enabled? && form.available_slots.positive? }
+        validates :reserved_slots, numericality: { greater_than_or_equal_to: 0 }, if: ->(form) { form.registrations_enabled? }
+        validates :reserved_slots, numericality: { less_than_or_equal_to: :available_slots }, if: ->(form) { form.registrations_enabled? }
 
+        validate :available_slots_greater_than_or_equal_to_registrations_count, if: ->(form) { form.registrations_enabled? && form.available_slots.positive? }
```

Comment:
The `if` part should be covered by previous validations, right? Should we drop it?

Reply:
are you sure? This validation existed, I just moved to other line